ios/PurchasesHybridCommon/PurchasesHybridCommon/Package+HybridAdditions.swift

```diff
@@ -9,8 +9,12 @@
 import Foundation
 import RevenueCat
 
-internal extension Package {
+public extension Package {
```

Comment:
Do we need to access these from the hybrids now?

Reply:
It's from `PurchasesHybridCommonUI` https://github.com/RevenueCat/purchases-hybrid-common/pull/735/files#diff-c673e3957e7bea968ef2fa452b8c46a98aac22c132cf33e54ab4ba6bbb02f025R200

Reply:
for example, CustomerInfo and Offerings are also public for the same reason :)

Reply:
Ohh right, I forgot 😅 Makes sense, thanks!